src/Game.js

```diff
@@ -9,7 +9,7 @@ const Round = require('./Round')
 class Game {
   constructor() {}
 
-  printMessage(deck, round) {
+  printMessage(deck) {
```

Comment:
I think you missed changing where this was used.

Reply:
Removed `round` param from line 29.